templates/procuration/_request_reasons.html.twig

```diff
@@ -0,0 +1,17 @@
+<div class="profile-value" id="request-reason">
+    {% if request.reason == constant('REASON_RESIDENCY', request) %}
```

Comment:
What do you think to refactor this elseif suite in one line with translations, for example :
`('procuration.request.reason.' ~ request.reason)|trans`

Reply:
what is the reason is unknown or not translated ? the current logic does not provide a fallback label

Reply:
This is a choice with a constraint, no need for a fallback.

Reply:
Good call, done in https://github.com/EnMarche/en-marche.fr/pull/1977/commits/b5fe1fd7370a3e893082c5b6fb6d2e0fdab5c53e.